jwst/wfs_combine/tests/test_wfs_combine.py

```diff
@@ -89,8 +111,157 @@ def test_create_combined(_jail, wfs_association,
         im1.save(path1)
         im2.save(path2)
 
-    result = WfsCombineStep.call(path_asn, do_refine=False)
+    result = WfsCombineStep.call(path_asn, do_refine=False, flip_dithers=False, psf_size=50,
```

Comment:
The `WfsCombineStep` class is no longer directly imported above, just the `wfs_combine` module. So does this need to be `result = wfs_combine.WfsCombineStep.call(...)`?

Reply:
I'm not clear about the issue. This is working.

Reply:
There's the `wfs_combine` submodule and the `wfs_combine` subpackage, both named the same. This is causing confusion I think.

Reply:
Ah, OK. So I withdraw my original comment.